### PR TITLE
PP-1103: Pbs_spawn fails when user installed python path is used

### PR DESCRIPTION
**Pull request by Jitendra-Rajak**
#### Issue-ID

* **[PP-1103](https://pbspro.atlassian.net/browse/PP-1103)**

#### Problem description
* Pbs_spawn fails when user installed python path is used

#### Cause / Analysis
* pbs_python executable was not looking for system python
#### Solution description
* system installed python path and PYTHON_HOME are set.

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [x] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by bhroam** [comment]
@Jitendra-Rajak Did you mean to raise a pull request against the 14.1 branch?

**Comment by subhasisb** [comment]
@bhroam yes Jitendra wanted to do the PR against the release_14_1_branch since we are poised to make a new release on that branch soon called 14.1.1 (14.1.0 is already tagged on that branch, so we are good to proceed).

**Comment by bremanandjk** [comment]
Change looks good & identical on comparing original commits; Travis test executions are all pass. Hence retaining original author name is fine with this case. I sign-off.

File: test/fw/ptl/lib/pbs_testlib.py [hunk 1]
```diff
@@ -4213,11 +4213,53 @@ class Comm(PBSService):
     PBS Comm configuration and control
     """
```
Comment (saritakh) [inline test/fw/ptl/lib/pbs_testlib.py]: Remove this line of extra quotes.

File: test/fw/ptl/lib/pbs_testlib.py [hunk 2]
```diff
@@ -4213,11 +4213,53 @@ class Comm(PBSService):
...
 
-    logger = logging.getLogger(__name__)
-    logprefix = 'pbs_comm: '
+    """
```
Comment (saritakh) [inline test/fw/ptl/lib/pbs_testlib.py]: Remove this line of extra quotes

File: test/fw/ptl/lib/pbs_testlib.py [hunk 3]
```diff
@@ -4213,11 +4213,53 @@ class Comm(PBSService):
...
+    :type diagmap: dictionary
+    :param diag: path to PBS diag directory (This will override diagmap)
+    :type diag: str or None
+    :param server: A PBS server instance to which this Comm is associated
+    :type server: str
+    :param db_access: set to either file containing credentials to DB access or
+                      dictionary containing {'dbname':...,'user':...,
+                      'port':...}
+    :type db_access: str or dictionary
+        """
```
Comment (saritakh) [inline test/fw/ptl/lib/pbs_testlib.py]: Improper indentation of the quotes.

File: test/tests/pbs_python_test.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+from subprocess import Popen,PIPE
+
+class Test_pbs_python(PBSTestSuite):
+	"""
+	This test suite tests pbs_python executable
+	and makes sure it works fine.
+	"""
+	def test_pbs_python(self):
+		"""
+		Thi method spawns a python process using
```
Comment (saritakh) [inline test/tests/pbs_python_test.py]: Update to "This".

**Comment by saritakh** [comment]
changes look good to me. I sign off.

**Comment by subhasisb** [comment]
All looks good. Will merge when Travis passes. We will ignore codacy and Appveyor for the 14 branch.

**Pull request merged**